backend/tests/services/execution/test_request_builder_task_data.py

```
@@ -0,0 +1,154 @@
+    }
+    defaults.update(overrides)
+    return ExecutionRequest(**defaults)
+
+
+class TestTaskTokenProperty:
+    """Tests for the task_token property alias on ExecutionRequest."""
+
+    def test_task_token_aliases_auth_token(self) -> None:
+        """task_token property should return auth_token value."""
+        request = _make_request(auth_token="my-auth-token-xyz")  # noqa: S106
+        assert request.task_token == "my-auth-token-xyz"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "task_token"

(S105)